Document that files:transfer-ownership changes in 10.4.1 (#2523)
owncloud/docs: modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc

== The files:transfer-ownership command

> You may transfer all files and shares from one user to another, including users who have never logged in. 

Reviewer: ```suggestion
You may transfer all files and shares from one user to another, including to a user who has never logged in yet. 
```
Maybe it will be clearer to say it like this ^
There is no point trying to transfer `from` a user who has never logged in, because that user will not have any files!

Author: It is clearer, yes. Thanks.
